pyface/ui/qt4/tasks/tests/test_split_editor_area_pane.py

```diff
@@ -321,6 +326,7 @@ def test_persistence(self):
         self.assertEquals(right_bottom.items[0].id, 1)
         self.assertEquals(right_bottom.items[1].id, 2)
 
+    @skip_if_no_traitsui
```

Comment:
`ViewWithTabsEditor` should be rewritten to not use `traitsui` instead of skipping these tests. It should be straightforward to do in Qt. Probably even shorter.